esmvalcore/cmor/fix.py

```diff
@@ -164,16 +189,28 @@ def fix_metadata(cubes,
             cube_list = fix.fix_metadata(cube_list)
 
         cube = _get_single_cube(cube_list, short_name, project, dataset)
-        checker = _get_cmor_checker(frequency=frequency,
-                                    table=project,
-                                    mip=mip,
-                                    short_name=short_name,
-                                    check_level=check_level,
-                                    fail_on_error=False,
-                                    automatic_fixes=True)
+
+        # Automatic fixes
+        automatic_fixer = AutomaticFix.from_facets(
+            project, mip, short_name, frequency=frequency
+        )
```

Comment:
Very good point, I haven't thought of this. Will try to change that! But please note that this will introduce a lot more changed files since I need to adapt each `get_fixes` call in the tests (and there are a lot of them!).

Reply:
Done in https://github.com/ESMValGroup/ESMValCore/commit/6954d2168280dd2ccbca0e782815d99041072dd4